File: domain/src/main/java/com/github/anrimian/musicplayer/domain/models/equalizer/EqualizerState.java
package com.github.anrimian.musicplayer.domain.models.equalizer;

import java.util.Map;

public class EqualizerState {

    private short currentPreset;
    private Map<Short, Short> bendLevels;

    public EqualizerState(short currentPreset, Map<Short, Short> bendLevels) {
        this.currentPreset = currentPreset;
        this.bendLevels = bendLevels;
    }

    public short getCurrentPreset() {
        return currentPreset;
    }

    public Map<Short, Short> getBendLevels() {
        return bendLevels;
    }

    public void setCurrentPreset(short currentPreset) {
        this.currentPreset = currentPreset;
    }

    public void setBendLevels(Map<Short, Short> bendLevels) {
        this.bendLevels = bendLevels;
    }

    @Override
    public String toString() {
        return "EqualizerState{" +
                "currentPreset=" + currentPreset +
                ", bendLevels=" + bendLevels +
                '}';
    }
}

File: domain/src/main/java/com/github/anrimian/musicplayer/domain/models/equalizer/EqualizerState.kt
package com.github.anrimian.musicplayer.domain.models.equalizer

class EqualizerState(var currentPreset: Short, var bandLevels: MutableMap<Short, Short>) {

    override fun toString(): String {
        return "EqualizerState{" +
                "currentPreset=" + currentPreset +
                ", bandLevels=" + bandLevels +
                '}'
    }
}
